index.html

```diff
@@ -0,0 +1,159 @@
+                <a class="link" href="#"> Sponsor</a>
+            </li>
+            <li>
+                <a class="link" href="#"> News</a>
+            </li>
+        </ul>
+    </div>
+    <main>
+        <section id="introduction">
+            <h3 class="introduction-greet">"Hello! Data Scientists"</h3>
+            <h1 class="introduction-h1">AI & BIG DATA FOR A SMARTER FUTURE EXHIBITION 2023</h1>
+            <P class="introduction-para">
+                Data literacy is so critical to our education system but also
+                key to humanity. This Exhibition will showcase the next generation
+                enterprise technologies and strategies from the world of Data Science
+                and AI, providing an opportunity to explore and discover the practical
+                and successful implementation of data science and machine learning to 
+                drive your business forward.
+            </P>
+            <div class="introduction-div">
+                <h2 class="intro-div-h2">2023.01.19(THUR) ~ 20(FRI)</h2>
+                <p class="intro-div-para">@ Riverside Drive, Nairobi, Kenya.</p>
+            </div>
+        </section>
+        <section id="program-section">
+            <h2>Main Program</h2>
+            <hr>
+            <div id="programs">
+                <div class="program">
+                    <img src="images/lecture.png" alt="Lecture">
+                    <h4>Lecture</h4>
+                    <p>Listen to the speakers from various countries about securing value through AI & Big Data.</p>
+                </div>
+                <div class="program">
+                    <img src="images/exhibition.png" alt="Exhibition">
+                    <h4>Exhibition</h4>
+                    <p>Appreciate various creations applying successful implementation of data science and AI.</p>
+                </div>
+                <div class="program">
+                    <img id="" src="images/forum.png" alt="Forum">
+                    <h4>Forum</h4>
+                    <p>Have time to share your thoughts and opinions with experts from each topic.</p>
+                </div>
+                <div class="program">
+                    <img src="images/workshop.png" alt="Workshop">
+                    <h4>Workshop</h4>
+                    <p>Try creating your own work using open source data science software rather than just watching it.</p>
+                </div>
+                <div class="program">
+                    <img src="images/ignite.png" alt="Ignite">
+                    <h4>Ignite</h4>
+                    <p>Get opportunities to network with numerous clients, prospects, and partners from around the world.</p>
+                </div>
+            </div>
+            <button type="button" class="program-btn">Join AI & Big Data Expo 2023</button>
+            <p class="whole-program">
+                <a href="#">SEE THE WHOLE PROGRAM</a>
+            </p>
+        </section>
```

Comment:
Kindly reduce the height, font weight, and font size, and also make sure that all the parr for every single program card part is exactly the size as defined in the original design as also suggested by the previous reviewer, especially on the mobile screen.😎🙏

Reply:
@Lizdev-05 Thank you for the feedback. By parr do you mean paragraph?